css/includes/components/itilobject/_timeline.scss

```diff
@@ -531,11 +531,12 @@ ul.legacy-timeline-actions {
     list-style: none;
     display: inline-flex;
     margin-bottom: 0;
+    position: relative;
+    top: -3px;
 
     li {
         display: inline;
         margin-left: 5px;
-        border: $input-border-width solid $input-border-color;
```

Comment:
can you explain this part ?

Reply:
If I keep the current CSS on GLPI master and use a button in Formcreator (see the linked PR), I get this :

![image](https://user-images.githubusercontent.com/14139801/151174902-acd2822d-0a72-4b02-aef6-2831d37d1303.png)

I think the border should be on the button itself, instead of the parent LI tag.

Reply:
ok